aws/cloudformation-templates/base/cloudfront.yaml

```diff
@@ -29,6 +22,13 @@ Resources:
       WebsiteConfiguration:
         IndexDocument: index.html
 
+  # Empties bucket when stack is deleted
+  EmptyWebUIBucket:
+    Type: Custom::EmptyStackBucket
+    Properties:
+      ServiceToken: !Ref CleanupBucketLambdaArn
+      BucketName: !Ref WebUIBucket
+
```

Comment:
Turn out that we have already had this for the `WebUIBucket`. (That is why we never had the issue with WebUI)

I moved this to below the `WebUIBucket` so that it's more logical to read.